src/odemis/acq/stitching/_registrar.py

```diff
@@ -87,7 +89,8 @@ def __init__(self):
         # initialize grid to 1x1. The size will increase as new tiles are
         # added.
         self.nx = 1  # (int) total number of cols
-        self.registered_positions = [[None]]  # (None or tuple of 2 floats) list of calculated positions
+        self.registered_positions_px = [[None]]  # (None or tuple of 2 floats) list of calculated shifts in px
+        self.registered_positions_m = [[None]]  # (None or tuple of 2 floats) list of calculated positions in m
```

Comment:
Do you actually need this? Is this for better understanding for the user? I think returning `tile_positions` will just do fine in line 162

Reply:
Indeed, this doesn't seem used any where. Also, the comment seem to imply the difference between the two is just a coefficient factor, but one is relative to the center of the first tile, while the other one is in absolute coordinates. Maybe worth mentioning?